backend/coordnet/templates/pages/home.html

```diff
@@ -9,6 +9,13 @@
           href="{% static "coordination-network-circle-small.png" %}" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>coordination.network</title>
+    <script>
+      window.__COORDNET_CONFIG__ = {
+        apiUrl: "{{ api_url }}",
```

Comment:
This will probably escape any `<>"'&`, but that shouldn't be an issue, right?

Reply:
I think that's fine